src/xtgeo/grid3d/_grid_etc1.py

```diff
@@ -342,9 +337,9 @@ def get_ijk_from_points(
         self.ncol,
         self.nrow,
         self.nlay,
-        self._coordsv,
-        self._zcornsv,
-        self._actnumsv,
+        self._coordsv.ravel(),
+        self._zcornsv.ravel(),
```

Comment:
Was suddenly getting warnings about the shape did not match the swig binding here... not sure!